master_server.py

```diff
@@ -47,13 +54,21 @@ def main(args):
     """Master server main function."""
     servers, has_ui = create_servers(silent=args.silent,
                                      debug_mode=args.debug_mode)
+    broadcast_queues = {"FmpServer": queue.Queue(), "OpnServer": queue.Queue(),
+                        "LmpServer": queue.Queue(), "RfpServer": queue.Queue()}
     threads = [
-        threading.Thread(target=server.serve_forever)
+        threading.Thread(target=server.serve_forever, name=server.__class__.__name__,
+            args=(broadcast_queues[server.__class__.__name__], ))
```

Comment:
If you plan to use `server.__class__.__name__` then you should also use it in `broadcast_queues` rather than hardcoding the (dict's key) string.

Reply:
Done